src/core/caniuse.js

```diff
@@ -80,21 +85,25 @@ export async function run(conf) {
     </dd>`;
   headDlElem.append(...definitionPair.childNodes);
   await contentPromise;
-
-  // remove from export
   pub("amend-user-config", { caniuse: options.feature });
-  sub("beforesave", outputDoc => {
-    html.bind(outputDoc.querySelector(".caniuse-stats"))`
-      <a href="${featureURL}">caniuse.com</a>`;
-  });
+  if (options.removeOnSave) {
+    // Will remove the browser support cells.
+    headDlElem
+      .querySelectorAll(".caniuse-browser")
+      .forEach(elem => elem.classList.add("removeOnSave"));
+    sub("beforesave", outputDoc => {
+      html.bind(outputDoc.querySelector(".caniuse-stats"))`
+        <a href="${featureURL}">caniuse.com</a>`;
+    });
+  }
```

Comment:
Should we remove it entirely instead?

Reply:
For that case, the actual configuration option can be set to null.